data_steward/analytics/cdr_ops/run_rdr_qc_report.py

```diff
@@ -5,25 +5,25 @@
 import papermill as pm
```

Comment:
You can remove this file, as this functionality will be handled by the report runner.

Reply:
i don't see this line in my updated file.

Reply:
i see. we can forget about this file. i used runner_report.

Reply:
i removed this file.

Reply:
Thanks